tests/test_TRDIpd0_analyzepd0file.py

```diff
@@ -4,14 +4,16 @@
 # these tests are designed to follow the example here
 # http://doc.pytest.org/en/latest/fixture.html
 import pytest
+from pathlib import Path
 
+here = Path(__file__).parent
```

Comment:
This is to allow us to find this file without relying on relative paths.
(We need it b/c the tests can be run from another place, like what I did above.)